web/src/FatalErrorModal.tsx

```diff
@@ -4,7 +4,7 @@ import { ShowFatalErrorModal } from "./types"
 import "./FatalErrorModal.scss"
 
 type props = {
-  error: string | null
+  error: string | null | undefined
```

Comment:
swagger generates typescript files with undefined instead of null for some reason?